File: Objective-C Version/KYCuteView/KYCuteView/ViewController.m
//
//  ViewController.m
//  KYCuteView
//
//  Created by Kitten Yang on 1/23/16.
//  Copyright © 2016 Kitten Yang. All rights reserved.
//

#import "ViewController.h"
#import "KYCuteView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    KYCuteView *cuteView = [[KYCuteView alloc]initWithPoint:CGPointMake(25, [UIScreen mainScreen].bounds.size.height - 65) superView:self.view];
    cuteView.viscosity  = 20;
    cuteView.bubbleWidth = 35;
    cuteView.bubbleColor = [UIColor colorWithRed:0 green:0.722 blue:1 alpha:1];
    [cuteView setUp];
    
    //注意：设置 'bubbleLabel.text' 一定要放在 '-setUp' 方法之后
    //Tips:When you set the 'bubbleLabel.text',you must set it after '-setUp'
    cuteView.bubbleLabel.text = @"13";
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: Swift Version/KYCuteView-Swift/KYCuteView-Swift/ViewController.swift
//
//  ViewController.swift
//  KYCuteView-Swift
//
//  Created by Kitten Yang on 1/19/16.
//  Copyright © 2016 Kitten Yang. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        var option = BubbleOptions()
        option.viscosity = 20.0
        option.bubbleWidth = 35.0
        option.bubbleColor = UIColor(red: 0.0, green: 0.722, blue: 1.0, alpha: 1.0)
        
        let cuteView = CuteView(point: CGPointMake(25, UIScreen.mainScreen().bounds.size.height - 65), superView: view, options: option)
        option.text = "20"
        cuteView.bubbleOptions = option

    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()

    }


}

